sickbeard/classes.py

```diff
@@ -212,7 +212,7 @@ def selectSeries(allSeries):
 
 
 class Proper(object):
-    def __init__(self, name, url, date, show, seeders, leechers, size, pubdate, hash, proper_tags):
+    def __init__(self, name, url, date, show, seeders, leechers, size, pubdate, torrent_hash, proper_tags):
```

Comment:
If we replaced it with torrent_hash somewhere else, never mind. It would just be the correct naming.